app/ante/min_fee_test.go

```diff
@@ -29,67 +39,88 @@ func TestCheckTxFeeWithGlobalMinGasPrices(t *testing.T) {
 
 	feeAmount := int64(1000)
 
-	ctx := sdk.Context{}
-
-	globalMinGasPrice := appconsts.DefaultGlobalMinGasPrice
-	require.NoError(t, err)
+	paramsKeeper, stateStore := setUp(t)
 
 	testCases := []struct {
 		name       string
 		fee        sdk.Coins
 		gasLimit   uint64
 		appVersion uint64
+		isCheckTx  bool
 		expErr     bool
 	}{
 		{
 			name:       "bad tx; fee below required minimum",
 			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount-1)),
-			gasLimit:   uint64(float64(feeAmount) / globalMinGasPrice),
+			gasLimit:   uint64(float64(feeAmount) / v2.GlobalMinGasPrice),
 			appVersion: uint64(2),
+			isCheckTx:  false,
 			expErr:     true,
 		},
 		{
 			name:       "good tx; fee equal to required minimum",
 			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount)),
-			gasLimit:   uint64(float64(feeAmount) / globalMinGasPrice),
+			gasLimit:   uint64(float64(feeAmount) / v2.GlobalMinGasPrice),
 			appVersion: uint64(2),
+			isCheckTx:  false,
 			expErr:     false,
 		},
 		{
 			name:       "good tx; fee above required minimum",
 			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount+1)),
-			gasLimit:   uint64(float64(feeAmount) / globalMinGasPrice),
+			gasLimit:   uint64(float64(feeAmount) / v2.GlobalMinGasPrice),
 			appVersion: uint64(2),
+			isCheckTx:  false,
 			expErr:     false,
 		},
 		{
 			name:       "good tx; with no fee (v1)",
 			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount)),
-			gasLimit:   uint64(float64(feeAmount) / globalMinGasPrice),
+			gasLimit:   uint64(float64(feeAmount) / v2.GlobalMinGasPrice),
 			appVersion: uint64(1),
+			isCheckTx:  false,
 			expErr:     false,
 		},
 		{
 			name:       "good tx; gas limit and fee are maximum values",
 			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, math.MaxInt64)),
 			gasLimit:   math.MaxUint64,
 			appVersion: uint64(2),
+			isCheckTx:  false,
 			expErr:     false,
 		},
 		{
 			name:       "bad tx; gas limit and fee are 0",
 			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, 0)),
 			gasLimit:   0,
 			appVersion: uint64(2),
+			isCheckTx:  false,
 			expErr:     false,
 		},
 		{
 			name:       "good tx; minFee = 0.8, rounds up to 1",
 			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount)),
 			gasLimit:   400,
 			appVersion: uint64(2),
+			isCheckTx:  false,
+			expErr:     false,
+		},
+		{
+			name:       "good tx; fee above node's required minimum",
+			fee:        sdk.NewCoins(sdk.NewInt64Coin(appconsts.BondDenom, feeAmount+1)),
+			gasLimit:   uint64(float64(feeAmount) / appconsts.DefaultGlobalMinGasPrice),
+			appVersion: uint64(1),
+			isCheckTx:  true,
 			expErr:     false,
 		},
```

Comment:
[question] would it be helpful to include a test case where a node operators locally configured min gas price differs from the constant at appconsts.DefaultGlobalMinGasPrice?